WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/BlogDashboardDynamicCardCell.swift

```diff
@@ -120,7 +120,16 @@ private extension DynamicDashboardCard {
             return .init(title: title, callback: callback)
         }()
 
-        let input = Input(featureImageURL: featureImageURL, rows: rows, action: action)
+        let input = Input(
+            featureImageURL: featureImageURL,
+            featureImageWidthToHeightRatio: CGFloat(
+                payload.featuredImageSize?.width ?? 2
+            ) / CGFloat(
+                payload.featuredImageSize?.height ?? 1
+            ),
```

Comment:
We could add a computed variable on `ImageSize` and just do this:
```suggestion
            featureImageWidthToHeightRatio: payload.featuredImageSize?.widthToHeightRatio ?? 2,
```